test/api/prompt.test.js

```diff
@@ -63,4 +75,23 @@ describe("prompt", () => {
 
         expect(result).toBe(false);
     });
+
+    it("should respond to SIGINT", async () => {
+        const runAndKillPrompt = resolve(__dirname, "./helpers/runAndKillPrompt.js");
+
+        const { exitCode, stderr, stdout } = await execa("node", [runAndKillPrompt], {
+            cwd: resolve(__dirname),
+            reject: false,
+            maxBuffer: Infinity,
+        });
+
+        // TODO: fix for windows
+        if (isWindows) {
```

Comment:
I think it is not good to skip windows test here...

Reply:
I will see if I can test it on a windows machine locally.